aries_cloudagent/protocols/out_of_band/v1_0/tests/test_manager.py

```diff
@@ -361,18 +362,20 @@ def setUp(self):
                 "debug.auto_accept_requests": True,
             }
         )
-        self.session.context.injector.bind_instance(BaseResponder, self.responder)
+        self.session = self.profile.session()
```

Comment:
Is this still needed?

Reply:
Yes, the session is created before every test and used in them, since It is often required in the setup stage